cmd/gitopiad/cmd/genesis.go

```diff
@@ -46,24 +49,59 @@ const (
 )
 
 const (
-	rewardsServiceAddress            = "gitopia1a875smmd9va45tsx398prdzjtm5fg23mlzzgck"
-	strategicReserveAddress1         = ""
-	strategicReserveAddress2         = ""
-	strategicReserveAddress3         = ""
-	strategicReserveAddress4         = ""
-	strategicReserveAddress5         = ""
-	feegrantsAddress                 = ""
-	earlySupportersMultiSigAddress   = ""
-	strategicPartnersMultiSigAddress = ""
-	advisorsMultiSigAddress          = ""
-	teamMultiSigAddress              = ""
+	rewardsServiceAddress            = "gitopia1r7mhw4f6x73lap55st3yxn4u7tj4yt6qqwmlv5"
+	strategicReserveAddress1         = "gitopia1ryv3840cdfjhknqrjkvuhyn6h4zylwvq9ekdv9"
+	strategicReserveAddress2         = "gitopia12lehgzv7w7k8x0vwu9nsu9fvj9e80evhsu3jpf"
+	strategicReserveAddress3         = "gitopia17505m7wwzh28sf6dud2a8rd8qztlp4xvz5nqca"
+	strategicReserveAddress4         = "gitopia17urengx7kxrsche82cfxu3qutxnng97k04vy9a"
+	strategicReserveAddress5         = "gitopia1hcq6ejnxmsxaqpee7vh7mqjysgw2kp8sf4g4xm"
+	feegrantsAddress                 = "gitopia13ashgc6j5xle4m47kqyn5psavq0u3klmscfxql"
```

Comment:
where does the fee grants balance come from?

Reply:
https://github.com/gitopia/gitopia/blob/785e5ae1c569f2437d848871aac92b78c7f0ee2b/cmd/gitopiad/cmd/genesis.go#L870